.github/ISSUE_TEMPLATE/release_task.yaml

```
@@ -27,7 +27,9 @@ body:
     id: build-assets
     attributes:
       label: Create new assets for release
-      description: Build binary files to upload.
+      description: |
+        * Build binary files to upload.
+        * Sign modules and installers.
```

非常に細かいですが、趣旨としてはこう表すのかなと思いました。

```suggestion
        * Sign the modules and the installer.
```
* 一度に署名するmoduleは複数でよいが、installerは1つだから単数形になるはず
* どのモジュールとインストーラーに署名するかは特定できているので、定冠詞theがあった方が意味が通じやすいかも

---

修正しました。

確かにインストーラーは一つでした。